gateway/gateway-controller/pkg/config/xds_tls_test.go

```
@@ -0,0 +1,364 @@
+		template.ExtKeyUsage = []x509.ExtKeyUsage{x509.ExtKeyUsageClientAuth}
+	}
+
+	der, err := x509.CreateCertificate(rand.Reader, template, caCert, &priv.PublicKey, caKey)
+	require.NoError(t, err)
+
+	certPath = filepath.Join(dir, name+".crt")
+	keyPath = filepath.Join(dir, name+".key")
+
+	certOut, err := os.Create(certPath)
+	require.NoError(t, err)
+	defer certOut.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `certOut.Close` is not checked

(errcheck)